nifi-nar-bundles/nifi-rules-bundle/nifi-rules-processors/src/test/java/org/apache/nifi/rules/processors/TestRulesRecordProcessor.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.rules.processors;
+
+import com.google.common.collect.Lists;
```

Comment:
Do we need Guava here? Maybe use `Arrays.asList()` instead, since I don't think Guava is an explicit dependency of this module (must be transitive)